### PR TITLE
Answer

### PR DESCRIPTION
**Pull request by marnie-k**
Robin has joined David and I in out HW study group so our work will look very similar. Also - I had this passing all the tests, then none of the tests, and I think the problem is in my package.json because I really didn't change anything substantial in the code and Robin and David were getting theirs to pass ?!?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// fixing git problem
 const validateLineup = (lineup) =>{
     alllineupsgood = true;
```
Comment (carryegaulin31) [inline index.js:2]: you set your variable to return true therefore simply tricking your tests.  The rest of the code in your project was not valid nor was it used 
image not sure if this is showing but all of your code was underlined and stated that none was actively being used.

**Review by carryegaulin31 — COMMENTED** [review]
In addition 3 commit messages is not nearly enough and they are not precise need work on those
